### PR TITLE
post about rladies sp

### PR DESCRIPTION
**Pull request by drmowinckels**

### CODE REVIEW

**Comment by drmowinckels** [comment]
@beatrizmilz because I spent WAY too long doing this, I decided to just start it over as I completely messed up your last PR. 
I'll go through this now, wait for your approval and then get it merged. I'm so sorry for all the back and forwards!

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+---
+title: "Actions to increase diversity in the R-Ladies São Paulo community"
+author: 
+  - Beatriz Milz
+  - Haydee Svab
+  - Tatyane Paz Dominguez
+  - Ianni Muliterno
+  - Ana Paula Rocha
+date: '2023-05-06'
+slug: []
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:10]:
```suggestion
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+categories:
+  - r-ladies
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:12]:
```suggestion
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  - community
+lastmod: '2023-05-06T15:53:00+03:00'
+keywords: []
+description: ''
+comment: yes
+toc: yes
+autoCollapseToc: no
+contentCopyright: no
+reward: no
+mathjax: no
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:27]:
```suggestion
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+categories:
+  - r-ladies
+  - R-Ladies
+  - community
+  - latin america
+tags:
+  - latin america
+  - community
+lastmod: '2023-05-06T15:53:00+03:00'
+keywords: []
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:20]:
```suggestion
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+-   [Event: Introduction to R Workshop](https://rladies-sp.org/posts/2023-01-evento-oficina-intro-R/)
+
+![Picture of participants in a recent event.](logo.png)
+
+### **Affirmative actions**
+
+Seeking a greater diversity of participants in our events, we saved spots focusing on three groups (quotas):
+
+-   Black, Brown and Indigenous people;
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:153]:
```suggestion
-   Black, Brown and Indigenous people
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+-   mothers;
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:155]:
```suggestion
-   mothers
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+
+-   women and other gender minorities.
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:157]:
```suggestion
-   women and other gender minorities
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+
+To do so, two points are important to be discussed: what is intersectionality, and the Brazilian context.
+
+## What is intersectionality?
+
+The target audience of R-Ladies is gender minorities: cis women, trans women, trans men, non-binary people, and queer people. The main purpose of the community is to include these minority groups in the R community. However, for each of these groups, there are people who are part of other groups that may also suffer other oppressions arising from social markers of difference, such as race, sexual orientation, maternity, disabilities, age, etc. Therefore, the oppressions are not just based on a person's gender, but are interrelated and overlapping. This understanding is a concept called "intersectionality".
+
+According to [Bridie Taylor (2019)](https://www.womankind.org.uk/intersectionality-101-what-is-it-and-why-is-it-important/): "Intersectionality is the acknowledgment that **everyone has their own unique experiences of discrimination and oppression** and we must consider everything and anything that can marginalize people -- gender, race, social class, sexual orientation, physical ability, etc."
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:44]:
```suggestion
According to [Bridie Taylor (2019)](https://www.womankind.org.uk/intersectionality-101-what-is-it-and-why-is-it-important/): 

> "Intersectionality is the acknowledgment that **everyone has their own unique experiences of discrimination and oppression** and we must consider everything and anything that can marginalize people -- gender, race, social class, sexual orientation, physical ability, etc."
```

**Review by drmowinckels — COMMENTED** [review]
I think that is all from me- Mostly minor things to fix for the format.

Other than that, if you want, we are very happy to also have your Portuguese version of the post ready, if you want to share it :)

**Comment by beatrizmilz** [comment]
Hi Mo @drmowinckels ! Thank you. 
I will send the Portuguese version!
I would like to add a brief information at the beginning of the post, saying that it was written in April, and since then, we have received a grant from R Consortium that made it possible other things. But it will be brief

**Comment by beatrizmilz** [comment]
As soon as I'm finished, I will let you know! But probably today until the end of the day, I will commit the Portuguese version. We had already written it, but I'm reviewing if there is anything that was changed, and also adding alt text in images for portuguese.

**Comment by beatrizmilz** [comment]
@drmowinckels I think now is ok! Can you check if it is ok to be posted?

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
+---
+title: "Actions to increase diversity in the R-Ladies São Paulo community"
+author: 
+  - Beatriz Milz
+  - Haydee Svab
+  - Tatyane Paz Dominguez
+  - Ianni Muliterno
+  - Ana Paula Rocha
+date: '2023-10-02'
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:9]:
```suggestion
date: '2023-10-04
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.pt.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
+---
+title: "Ações para ampliar a diversidade na comunidade R-Ladies São Paulo"
+author: 
+  - Beatriz Milz
+  - Haydee Svab
+  - Tatyane Paz Dominguez
+  - Ianni Muliterno
+  - Ana Paula Rocha
+date: '2023-10-02'
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.pt.md:9]:
```suggestion
date: '2023-10-04'
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.pt.md [hunk 11]
```diff
@@ -0,0 +1,249 @@
...
+categories:
+  - R-Ladies
+  - community
+  - latin america
+tags:
+  - latin america
+  - community
+lastmod: '2023-10-02T10:00:00+03:00'
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.pt.md:17]:
```suggestion
```

File: content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+categories:
+  - R-Ladies
+  - community
+  - latin america
+tags:
+  - latin america
+  - community
+lastmod: '2023-10-02T10:00:00+03:00'
```
Comment (drmowinckels) [inline content/blog/2023-06-05-increasing-diversity-sao-paulo/index.en.md:17]:
```suggestion
```

**Review by drmowinckels — COMMENTED** [review]
Just fixing the date left, and then we should be ok :)

**Review by SoyAndrea — COMMENTED** [review]
everything is fine for me, beautiful post

**Comment by beatrizmilz** [comment]
> everything is fine for me, beautiful post

Hi Andrea! The post was merged.  https://rladies.org/blog/2023-06-05-increasing-diversity-sao-paulo/
It would be nice to share in social media!

**Pull request merged**